src/partclone.c

```diff
@@ -1883,7 +1883,7 @@ void print_partclone_info(cmd_opt opt) {
 		else
 		    log_mesg(0, 0, 1, debug, _("Starting to restore image (%s) to device (%s)\n"), opt.source, opt.target);
 	}else if(opt.dd)
-		log_mesg(0, 0, 1, debug, _("Starting to back up device(%s) to device(%s)\n"), opt.source, opt.target);
+		log_mesg(0, 0, 1, debug, _("Starting to back up device(%s) to device (%s)\n"), opt.source, opt.target);
```

Comment:
Wouldn't applying this consistently make even more sense?

```suggestion
		log_mesg(0, 0, 1, debug, _("Starting to back up device (%s) to device (%s)\n"), opt.source, opt.target);
```

Reply:
Yes, you are right...
I not saw it.